src/stores/Object.js

```diff
@@ -95,6 +95,34 @@ class ObjectStore {
     Object.keys(response).forEach(permission => this.objects[objectId][permission] = response[permission]);
   });
 
+  @action.bound
+  AddContentAdminsGroupPermissions = flow(function * ({objectId}) {
+    try {
+      // Automatically add permissions for content manage
+      const contentAdminsGroupAddress = yield Fabric.GetContentAdminsGroupAddress();
+      yield Fabric.AddContentObjectGroupPermission({
+        objectId,
+        groupAddress: contentAdminsGroupAddress,
+        permission: "see"
+      });
+      yield Fabric.AddContentObjectGroupPermission({
+        objectId,
+        groupAddress: contentAdminsGroupAddress,
+        permission: "access"
+      });
```

Comment:
Here we only need to add Manage.  The 'access' and 'see' are subsumed by Manage.

(plus it's less transactions which is good)

Reply:
@elv-serban I thought that was only true for library group permissions?

Reply:
I checked with zenia and she said you need the 3. I'm fine with only adding manage if that works

Reply:
Ah nevermind, fabric browser does only add the one permission in that case